container/sagemaker/gunicorn_service.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
naming is weird... why do we have a gunicorn service and a python service that are both falcon apps? it seems like these should be part of same module.

Reply:
There's also a falcon app called model manager. I can put them all into the same module.